test/browser.js

```diff
@@ -0,0 +1,28 @@
+import path from 'path';
+import fs from 'fs';
+import test from 'ava';
+import execa from 'execa';
+import objectAssign from 'object-assign';
```

Comment:
Yup, core-js (used in Babel) polyfills all static methods.

https://github.com/zloirock/core-js#ecmascript-6-object